sridha2412/assignment2/q1.py

```diff
@@ -8,24 +8,26 @@ def __init__(self, data: int):
         self.left = None
         self.right = None
 
-def find_ancestors(root: Node, key: int) -> bool:
+def find_ancestors(root: Node, key: int, ancestors: list) -> bool:
```

Comment:
Have you thought about attaching the method to the class itself. It really looks ready for it: replace 'root' with 'self' and add two more spaces in front of each lines.